src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig

```diff
@@ -373,6 +373,8 @@
                         <div class="col-md-12">
                           <fieldset class="form-group">
                             <label class="form-control-label">{{ form.step3.virtual_product.file.vars.label }}</label>
+                            <span class="help-box" data-toggle="popover"
+                                  data-content="{{ "Upload a file from your computer %maxUploadSize% max.)"|trans({'%maxUploadSize%': max_upload_size}, 'Admin.Catalog.Help') }}" ></span>
```

Comment:
We're missing a parenthesis here:
Upload a file from your computer **(**%maxUploadSize% max.)

Thank you!